deltachat-ios/Helper/MediaPicker.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+import Photos
+import MobileCoreServices
+
+protocol MediaPickerDelegate: class {
+    func onMediaSelected(url: NSURL)
+    func onDismiss()
+}
+
+class MediaPicker: NSObject, UINavigationControllerDelegate, UIImagePickerControllerDelegate {
+    private let navigationController: UINavigationController
+    private weak var delegate: MediaPickerDelegate?
+
+    init(navigationController: UINavigationController) {
+        self.navigationController = navigationController
+    }
+
+    func showPhotoLibrary(delegate: MediaPickerDelegate) {
+        if PHPhotoLibrary.authorizationStatus() != .authorized {
+            PHPhotoLibrary.requestAuthorization { status in
+                DispatchQueue.main.async {
+                    [weak self] in
+                    switch status {
+                    case  .denied, .notDetermined, .restricted:
+                        print("denied")
+                    case .authorized:
+                        self?.presentPhotoLibrary(delegate: delegate)
+                    }
+                }
+            }
+        } else {
+            self.presentPhotoLibrary(delegate: delegate)
+        }
+    }
+
+    private func presentPhotoLibrary(delegate: MediaPickerDelegate) {
+        if UIImagePickerController.isSourceTypeAvailable(.photoLibrary) {
+            let photoPicker = UIImagePickerController()
+            photoPicker.title = String.localized("photo")
```

Comment:
This is not included by any xml file.

Reply:
ahh, i see. @cyBerta this should be either "camera" or "gallery" then.

Reply:
Other non-included strings may also exist. Please use genstrings. For example, on the backup screen, I saw the loading string that could not be translated.

Reply:
Thanks for the hints! I rebased onto master and fixed the string issue